### PR TITLE
feat: add option to use Geist Sans instead of Inter

### LINKED ISSUE #1683: feat: Giest instead of Inter
### Is your feature request related to a problem? Please describe.

I'm always frustrated when [Theo.gg makes a video](https://www.youtube.com/watch?app=desktop&v=pwbTN_vGmDs) about how he is so excited about Vercel's launch of [Giest](https://vercel.com/font) yet it is not supported by default in t3.

### Describe the solution you'd like to see

I would like to see `Inter` replaced with `GeistSans`

### Describe alternate solutions

`GeistMono` is also an option. But, I prefer sans.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kakeWasTaken**
Closes #1683 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Added the option in the CLI to create a project with Geist Sans as the default font instead of Inter.

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Review by juliusmarminge — CHANGES_REQUESTED** [review]
Hey there, thanks for filing a pull request to this project! Highly appreciate it.

I think we should just change the default to be Geist and not provide an option in the CLI. Want to keep the number of prompts to a minimum.

WDYT @c-ehrlich ?

**Comment by nexxeln** [comment]
why do we need geist as default? it's one extra package which needs to be installed and people who don't want to use geist need to then uninstall a package to change the font. i personally think inter is a great default

**Comment by kakeWasTaken** [comment]
Hey, I removed the option to choose a font and set Geist to be the default font instead. 

Let me know if I should change anything else.

**Comment by juliusmarminge** [comment]
can you update the PR - there's a lot of diff and seems like you're a couple commits behind main

**Review by kakeWasTaken — COMMENTED** [review]
Set default font to Geist Sans instead of Inter
Updated to the latest version of create-t3-app

File: cli/package.json [hunk 1]
```diff
@@ -94,5 +94,8 @@
     "type-fest": "^3.7.0",
     "typescript": "^5.4.2",
     "zod": "^3.23.3"
-  }
+  },
+  "trustedDependencies": [
+    "es5-ext"
```
Comment (juliusmarminge) [inline cli/package.json:99]: ????

File: cli/src/cli/index.ts [hunk 2]
```diff
@@ -331,6 +331,7 @@ export const runCli = async (): Promise<CliResults> => {
     if (project.authentication === "next-auth") packages.push("nextAuth");
     if (project.database === "prisma") packages.push("prisma");
     if (project.database === "drizzle") packages.push("drizzle");
+    packages.push("geist");
```
Comment (juliusmarminge) [inline cli/src/cli/index.ts:334]: Just add it in base/package.json instead

File: cli/src/installers/dependencyVersionMap.ts [hunk 3]
```diff
@@ -37,5 +37,8 @@ export const dependencyVersionMap = {
   "@tanstack/react-query": "^5.25.0",
   superjson: "^2.2.1",
   "server-only": "^0.0.1",
+
+  // Geist
+  geist: "^1.3.0",
```
Comment (juliusmarminge) [inline cli/src/installers/dependencyVersionMap.ts:42]: Just add it in base/package.json instead

File: cli/src/installers/geist.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { type Installer } from "~/installers/index.js";
+import { addPackageDependency } from "~/utils/addPackageDependency.js";
+
+export const geistInstaller: Installer = ({ projectDir }) => {
```
Comment (juliusmarminge) [inline cli/src/installers/geist.ts:4]: Need to handle tailwind/non-tailwind css variable/className?

File: cli/template/base/next.config.js [hunk 5]
```diff
@@ -17,6 +17,7 @@ const config = {
     locales: ["en"],
     defaultLocale: "en",
   },
+  transpilePackages: ["geist"],
```
Comment (juliusmarminge) [inline cli/template/base/next.config.js:20]: ?

Comment (kakeWasTaken) [reply cli/template/base/next.config.js:20]: I think there was an issue with Geist, but without this line there would be an issue when using the pages router. 
Check this issue: https://github.com/vercel/geist-font/issues/13

File: cli/package.json (hunk 1 above)
Comment (kakeWasTaken) [reply cli/package.json:99]: idk, it was automatically added by my package manager

Comment (kakeWasTaken) [reply cli/package.json:99]: will remove

File: cli/src/cli/index.ts (hunk 2 above)
Comment (kakeWasTaken) [reply cli/src/cli/index.ts]: fixed in latest commit

File: cli/src/installers/dependencyVersionMap.ts (hunk 3 above)
Comment (kakeWasTaken) [reply cli/src/installers/dependencyVersionMap.ts]: fixed in latest commit

File: cli/template/extras/config/tailwind.config.ts [hunk 6]
```diff
@@ -1,12 +1,11 @@
 import { type Config } from "tailwindcss";
-import { fontFamily } from "tailwindcss/defaultTheme";
 
 export default {
   content: ["./src/**/*.tsx"],
   theme: {
     extend: {
       fontFamily: {
-        sans: ["var(--font-sans)", ...fontFamily.sans],
```
Comment (juliusmarminge) [inline cli/template/extras/config/tailwind.config.ts:9]: why remove all the default fallbacks?

**Comment by juliusmarminge** [comment]
Linter fails

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**